### PR TITLE
add alerta service

### PR DESCRIPTION
**Pull request by Arubinu**
# Description

Add Alerta.io in Notification Services.
But the service does not show up, as shown on this number: https://github.com/louislam/uptime-kuma/issues/1186

## Type of change

Please delete any options that are not relevant.

- Alerta notification service
- User interface (UI)
- Translation update

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I ran ESLint and other linters for modified files
- [X] I have performed a self-review of my own code and tested it
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

### CODE REVIEW

File: server/notification-providers/alerta.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+
+        try {
+            let alertaUrl = `${notification.alertaapiEndpoint}/alert`;
+            let config = {
+                headers: {
+                    "Content-Type": "application/json",
+                    "Authorization": "Key " + notification.alertaapiKey,
+                    "Accept": "text/json"
```
Comment (Saibamen) [inline server/notification-providers/alerta.js:17]:
```suggestion
                    "Accept": "text/json",
```

File: server/notification-providers/alerta.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                status: notification.alertarecoverState, // cleared
+                correlate: [],
+                service: [ "UptimeKuma" ],
+                group: "UptimeKuma-" + monitorJSON["type"], // Type de Sonde
+                value: "Timeout",
+                tags: [ "uptimekuma" ],
+                attributes: {},
+                origin: "uptimekuma",
+                type: "browserAlert",
+                resource: monitorJSON["name"]
```
Comment (Saibamen) [inline server/notification-providers/alerta.js:32]:
```suggestion
                resource: monitorJSON["name"],
```

File: server/notification.js [hunk 3]
```diff
@@ -65,7 +66,8 @@ class Notification {
             new SerwerSMS(),
             new Stackfield(),
             new WeCom(),
-            new GoogleChat()
+            new GoogleChat(),
+            new Alerta()
```
Comment (Saibamen) [inline server/notification.js:70]:
```suggestion
            new Alerta(),
```

File: src/components/notifications/index.js [hunk 4]
```diff
@@ -60,7 +61,8 @@ const NotificationFormList = {
     "serwersms": SerwerSMS,
     "stackfield": Stackfield,
     "WeCom": WeCom,
-    "GoogleChat": GoogleChat
+    "GoogleChat": GoogleChat,
+    "alerta": Alerta
```
Comment (Saibamen) [inline src/components/notifications/index.js:65]:
```suggestion
    "alerta": Alerta,
```

File: src/components/notifications/Alerta.vue [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+        <label for="alerta-api-endpoint" class="form-label">{{ $t("apiEndpoint") }}</label>
+        <input id="alerta-api-endpoint" v-model="$parent.notification.alertaapiEndpoint" type="text" class="form-control" required>
+        <label for="alerta-environment" class="form-label">{{ $t("environment") }}</label>
+        <input id="alerta-environment" v-model="$parent.notification.alertaenvironment" type="text" class="form-control" required>
+        <label for="alerta-api-key" class="form-label">{{ $t("apiKey") }}</label>
+        <input id="alerta-api-key" v-model="$parent.notification.alertaapiKey" type="text" class="form-control" required>
+        <label for="alerta-alert-state" class="form-label">{{ $t("alertState") }}</label>
+        <input id="alerta-alert-state" v-model="$parent.notification.alertaalertState" type="text" class="form-control" required>
+        <label for="alerta-recover-state" class="form-label">{{ $t("recoverState") }}</label>
+        <input id="alerta-recover-state" v-model="$parent.notification.alertarecoverState" type="text" class="form-control" required>
```
Comment (Saibamen) [inline src/components/notifications/Alerta.vue:12]: Fix camel case

Example: `alertaapiEndpoint` -> `alertaApiEndpoint`
`alertaenvironment` -> `alertaEnvironment`

**Review by Saibamen — CHANGES_REQUESTED** [review]

**Comment by Arubinu** [comment]
Wait, if I made this PR, it's because I was asked to help me with my problem.

**Comment by Arubinu** [comment]
We had to make: "npm run test-with-build".
But it took me a long time to find. It is not explained anywhere that everything is downloaded, and therefore it does not take the sources into account.

**Comment by Arubinu** [comment]
I'm fixing the issues before resubmitting a working PR.

**Comment by chakflying** [comment]
Sorry for your experience, but the [contributing guide](https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md) should be pretty clear on the procedure needed to start the development server, which should not use the pre-built frontend files.

I'd also recommend prefixing your new language keys with `alerta` as well.

**Pull request closed without merging**